frameworks/g-cloud-9/manifests/declaration.yml

```diff
@@ -137,8 +140,9 @@
   slug: application-accuracy
   editable: True
   prefill: False
-  description: >
-    You must confirm the accuracy of the information in this declaration and in your service descriptions. We might ask for proof after you’ve submitted your application.
+  description: |
+    You must confirm the accuracy of the information in this declaration and in your service descriptions. We might 
+    ask for proof after you’ve submitted your application.
```

Comment:
How do you mean?  

It should be `description: | You must confirm...` rather than dropping it down?